geps/gep-713.md

```diff
@@ -38,24 +58,211 @@ portable.
+      - name: CAcert
+    port: 8443
+
+```
+
+All this does is tell an implementation, that for connecting to port `8443` on the
+Service `fooService`, it should assume that the connection is TLS, and expect the
+service's certificate to be validated by the chain in the `CAcert` Secret.
+
+Importantly, this would apply to _every_ usage of that Service across any HTTPRoutes
+in that namespace, which could be useful for a Service that is reused in a lot of
+HTTPRoutes.
+
+With these two examples in mind, here are some guidelines for when to consider
+using Direct Policy Attachment:
+
+* The number or scope of objects to be modified is limited or singular. Direct
+  Policy Attachments should target one specific object (preferred), or a tightly-scoped
+  set of objects (like all Services in a namespace).
+* The modifications to be made to the objects don’t have any transitive information -
+  that is, the modifications only affect the single object that the targeted
+  metaresource is bound to, and don’t have ramifications that flow beyond that
+  object.
+* In terms of status, it should be reasonably easy for a user to understand that
+  everything is working - basically, as long as the targeted object exists, and
+  the modifications are valid, the metaresource is valid, and this should be
+  straightforward to communicate in one or two Conditions. Note that at the time
+  of writing, this is *not* completed.
+* Direct Policy Attachment _should_ only be used to target objects in the same
+  namespace as the Policy object. Allowing cross-namespace references brings in
+  significant security concerns, and/or difficulties about merging cross-namespace
+  policy objects. Notably, Mesh use cases may need to do something like this for
+  consumer policies, but in general, Policy objects that modify the behavior of
+  things outside their own namespace should be avoided unless it uses a handshake
+  of some sort, where the things outside the namespace can opt–out of the behavior.
+  (Notably, this is the design that we used for ReferenceGrant).
+
+### Inherited Policy Attachment: It's all about the defaults and overrides
+
+Because a Inherited Policy is a metaresource, it targets some other resource
+and _augments_ its behavior.
+
+But why have this distinct from other types of metaresource? Because Inherited
+Policy resources are designed to have a way for settings to flow down a hierarchy.
+
+Defaults set the default value for something, and can be overridden by the
+“lower” objects (like a connection timeout default policy on a Gateway being
+overridable inside a HTTPRoute), and Overrides cannot be overridden by “lower”
+objects (like setting a maximum client timeout to some non-infinite value at the
+Gateway level to stop HTTPRoute owners from leaking connections over time).
+
+Here are some guidelines for when to consider using a Inherited Policy object:
+* The settings or configuration are bound to one containing object, but affect
```

Comment:
Newline needed for markdown list rendering: https://deploy-preview-1565--kubernetes-sigs-gateway-api.netlify.app/references/policy-attachment/

```suggestion

* The settings or configuration are bound to one containing object, but affect
```